docker/lite/Dockerfile

```diff
@@ -51,16 +55,19 @@ RUN groupadd -r vitess && useradd -r -g vitess vitess
 RUN mkdir -p /vt/vtdataroot && chown -R vitess:vitess /vt
 
 # Set up Vitess environment (just enough to run pre-built Go binaries)
-ENV VTROOT /vt/src/vitess.io/vitess
```

Comment:
The only use was for the `vtdataroot` directory, but we might as well put everything under `/vt` instead of having two different directories.